manifests/install.pp

```diff
@@ -40,10 +40,20 @@
     }
   }
 
-  $file = "atlassian-${jira::product}-${jira::version}.${jira::format}"
+  # https://www.atlassian.com/software/jira/downloads/binary/atlassian-jira-software-7.0.0-jira-7.0.0-x64.bin
+
+  $arr_version = split($jira::version, '.')
+
+  $major = 0 + $arr_version[0]
```

Comment:
This is a way to convert the string into a number. But this isn't  the last code. 

Int the end I just used $jira::version[0] 

Let me see why it came to you